### DISCUSSION TITLE
Mock same type of interface twice in Mockito

### DISCUSSION THREAD

**Discussion by mepox** [trebol-ecommerce/trebol-backend-monolith#167] [Q&A] [answered]
I am writing tests for `SalesProcessServiceImpl`, everything was fine so far, reached coverage 77%. 

But to test the rest of the code I need to @Mock an other `ITwoWayConverterJpaService` however Mockito goes mad when I try to do it. 😄 

```
@Mock	ITwoWayConverterJpaService<SellPojo, Sell> converterService;
@Mock	ITwoWayConverterJpaService<ProductPojo, Product> productConverterService;
```

Any ideas?

**Comment by NyorJa** [comment, marked as answer]
Let me check. Please post the PR

**Reply by mepox** [reply]
> Let me check. Please post the PR

Hi @NyorJa ! Thanks for the reply. #168
